partcad/src/partcad/__init__.py

```diff
@@ -84,17 +91,77 @@ def before_send(event, hint):
         return event
 
     sentry_sdk.init(
+        # FIXME: sentry_sdk.utils.BadDsn: Missing public key
         dsn=user_config.get_string("sentry.dsn"),
         release=__version__,
+        environment=user_config.get_string("sentry.environment"),
         debug=user_config.get_bool("sentry.debug"),
         shutdown_timeout=user_config.get_int("sentry.shutdown_timeout"),
-        enable_tracing=True,
-        attach_stacktrace=False,
+        enable_tracing=user_config.get_bool("sentry.enable_tracing"),
+        attach_stacktrace=user_config.get_bool("sentry.attach_stacktrace"),
         traces_sample_rate=user_config.get_float("sentry.traces_sample_rate"),
         integrations=[
             LoggingIntegration(
-                level=logging.ERROR,
+                level=logging.logging.getLevelName(
+                    user_config.get_string("sentry.integrations.LoggingIntegration.level").upper()
+                )
             )
         ],
         before_send=before_send,
     )
+
+    # # https://develop.sentry.dev/sdk/data-model/event-payloads/user/
+    # # For some reason, the {{auto}} tag is not working:
+    # # "null - Replaced because of SDK configuration"
+    # sentry_sdk.set_user({"ip_address": "{{auto}}"})
+
+    sentry_sdk.set_tag("utm.source", "PartCAD")
+    sentry_sdk.set_tag("utm.medium", "Sentry")
+    sentry_sdk.set_tag("utm.campaign", "Open Source")
+    sentry_sdk.set_tag("utm.content", "API")
```

Comment:
This is for possible cases if we will have to provide custom versions for trials and then we can change `utm.campaign` and be able to filter only specific usages. It could be done other way tho, for example just loading additional tags from User Config.